platforms/jvm/plugins-java-base/src/integTest/groovy/org/gradle/api/plugins/jvm/internal/DefaultJvmFeatureTest.groovy

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2023 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.api.plugins.jvm.internal
+
+import org.gradle.api.InvalidUserDataException
+import org.gradle.api.plugins.JavaBasePlugin
+import org.gradle.api.plugins.JavaPluginExtension
+import org.gradle.test.fixtures.AbstractProjectBuilderSpec
+
+class DefaultJvmFeatureTest extends AbstractProjectBuilderSpec {
+    // TODO: This test definitely isn't appropriate for Component any more, so I've moved it here.
+    // Should features create the sourcesets they are going to use?  If they are being passed in
+    // like this, this is just testing the SourceSets container, not the feature.
+    def "cannot create multiple feature instances with the same source set"() {
```

Comment:
The most important part about the tests that you removed is that we are able to create multiple features in a single project -- since they definitely need to be able to live side-by-side. 

Can we add tests here to verify we can instantiate multiple features in the same project?

Reply:
Isn't that redundant?  We have lots of tests that already register multiple features on java projects.  `CapabilitiesConflictResolutionIssuesIntegrationTest`, `JavaLibraryFeatureCompilationIntegrationTest` for instance.

I can add an additional test here so it's easily findable and just has the simple goal of verifying multiple `DefaultJvmFeature`s can be created.  The tests I deleted were definitely about component interactions, not features, which is why I didn't worry about replacing any except for this one.

Reply:
Extra test added.